tedana/selection/_utils.py

```diff
@@ -58,7 +58,7 @@ def getelbow_cons(arr, return_val=False):
         (arr[nk - 5 - ii - 1] > arr[nk - 5 - ii : nk].mean() + 2 * arr[nk - 5 - ii : nk].std())
         for ii in range(nk - 5)
     ]
-    ds = np.array(temp1[::-1], dtype=np.int)
+    ds = np.array(temp1[::-1], dtype=np.integer)
```

Comment:
I think `numpy.int` is equivalent to the built-in `int`, rather than `numpy.integer`.

Reply:
@jbteves and I have been dealing with this in several places so I'll prod him to reply. We had `np.int` in multiple places in the code. At some minor numpy version change, that stopped working so a bunch have been switched to `np.integer` Not sure why this one got triggered here & not earlier. For the use cases we have, I'm not sure if it matters which one we use as long as it doesn't crash with a future version change.

Reply:
I think @jbteves opened #782 about this problem. The numpy documentation says that `numpy.int` is just an alias for `int`, though, so I think it makes more sense to replace `numpy.int` with `int` instead of `numpy.integer`. However, it's not a big deal, since I'm sure `numpy.integer` will work really well too.

Reply:
Actually, even though `np.int` _is_ closer to `int` than `np.integer`, whether that defaults to `int64` or `int32` depends on the system. It might be best for tedana to use `np.int32` across the board (not just in niimgs).